tracer/src/Datadog.Trace/Logging/DirectSubmission/Sink/DirectSubmissionLogSink.cs

```diff
@@ -11,10 +11,11 @@
 using System.Threading.Tasks;
 using Datadog.Trace.Logging.DirectSubmission.Formatting;
 using Datadog.Trace.Logging.DirectSubmission.Sink.PeriodicBatching;
+using Datadog.Trace.Telemetry;
 
 namespace Datadog.Trace.Logging.DirectSubmission.Sink
 {
-    internal class DatadogSink : BatchingSink, IDatadogSink
+    internal class DirectSubmissionLogSink : BatchingSink<DatadogLogEvent>, IDirectSubmissionLogSink
```

Comment:
Should `DatadogLogEvent` be renamed as well?

Reply:
👍 Yeah, makes sense, I'll change it to `DirectSubmissionLogEvent` instead

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/4402/commits/4c40e4c8c56de85806682ce22ba1045f317a7430